internal/core/unittest/test_index_wrapper.cpp

```diff
@@ -278,6 +278,7 @@ TEST(IVFFLATNMWrapper, Build) {
 }
 
 TEST(IVFFLATNMWrapper, Codec) {
+    int64_t flat_nb = 1000000;
```

Comment:
1000000 may be a very large number for ut. Consider moving this part to benchmark?

Reply:
Small data sets don't start with bugs.